packages/vue-components/src/panels/NestedPanel.vue

```diff
@@ -1,9 +1,18 @@
 <template>
   <span ref="cardContainer" :class="['card-container', addClass]">
     <div v-show="localMinimized" class="morph">
-      <button :class="['morph-display-wrapper', 'btn', btnType, 'card-title']" @click="open()">
+      <button
+        :class="['morph-display-wrapper', 'btn', btnType, 'card-title']"
+        @click="open()"
+        id="minimised-panel"
```

Comment:
hmm would the existing `morph-display-wrapper` class work instead? since we have multiple of these with the same `id`s

Reply:
Yes, it works and it is fixed.